### DISCUSSION TITLE
CMake Error While Insallation

### DISCUSSION THREAD

**Discussion by erentasken** [hyprwm/Hyprland#4947] [Q&A]
CMake Error at CMakeLists.txt:114 (add_executable):
  Cannot find source file:

    protocols/cursor-shape-v1-protocol.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm .h
  .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .hip .ispc

**Comment by erentasken** [comment]
make[1]: Leaving directory '/home/eren/Downloads/Hyprland'

**Reply by SamiKhan-cse19** [reply]
Did you solve the problem? I have the same issue.
